index.html

```diff
@@ -190,9 +199,32 @@ <h2>Remaining Work</h2>
     </section>
 
     <section>
-      <h2>Projects</h2>
+      <h2>Ecosystem Projects</h2>
       <ul>
-        <li>TODO: List ecosystem projects here</li>
+        <li>
+          <a href="https://github.com/rs-ipfs/rust-ipfs">rust-ipfs</a>
+          is an implementation that focuses on the suite of conformance
+          tests provided by Protocol Labs
+        </li>
+        <li>
+          <a href="#">rust-ipfs-api</a> - A HTTP client for an existing
+          go-ipfs or js-ipfs node. Supports both hyper and actix
+        </li>
+        <li>
+          <a href="#">ipfs-embed</a> - A light client implementation
+          based on sled
```

Comment:
```suggestion
          <a href="#">ipfs-embed</a> - A light client implementation
          with a <a href="https://github.com/spacejam/sled">sled</a> backend
```

I did not check if this is accurate but I think the wording is now better?